### ISSUE TITLE
active_shipping_calculator_spec fails

### ISSUE THREAD

**Issue by brchristian** [spree-contrib/spree_active_shipping#215]
When I clone the `2-3-stable` branch of `spree_active_shipping` and run the following:

```
bundle install
be rake test_app
be rspec spec/models/active_shipping_calculator_spec.rb
```

I am met with all specs failing as follows:

```
  1) Spree::Calculator::Shipping available when rates are available should return true
     Failure/Error: order = FactoryGirl.create(:order_with_line_items, :ship_address => address, :line_items_count => 2)
     ArgumentError:
       wrong number of arguments (1 for 0)
     # ./spec/models/active_shipping_calculator_spec.rb:11:in `block (2 levels) in <module:ActiveShipping>'

  2) Spree::Calculator::Shipping available when rates are available should use zero as a valid weight for service
     Failure/Error: order = FactoryGirl.create(:order_with_line_items, :ship_address => address, :line_items_count => 2)
     ArgumentError:
       wrong number of arguments (1 for 0)
     # ./spec/models/active_shipping_calculator_spec.rb:11:in `block (2 levels) in <module:ActiveShipping>'

  3) Spree::Calculator::Shipping available when rates are not available should return false
     Failure/Error: order = FactoryGirl.create(:order_with_line_items, :ship_address => address, :line_items_count => 2)
     ArgumentError:
       wrong number of arguments (1 for 0)
     # ./spec/models/active_shipping_calculator_spec.rb:11:in `block (2 levels) in <module:ActiveShipping>'

  4) Spree::Calculator::Shipping available when there is an error retrieving the rates should return false
     Failure/Error: order = FactoryGirl.create(:order_with_line_items, :ship_address => address, :line_items_count => 2)
     ArgumentError:
       wrong number of arguments (1 for 0)
     # ./spec/models/active_shipping_calculator_spec.rb:11:in `block (2 levels) in <module:ActiveShipping>'

  5) Spree::Calculator::Shipping compute should use the carrier supplied in the initializer
     Failure/Error: order = FactoryGirl.create(:order_with_line_items, :ship_address => address, :line_items_count => 2)
     ArgumentError:
       wrong number of arguments (1 for 0)
     # ./spec/models/active_shipping_calculator_spec.rb:11:in `block (2 levels) in <module:ActiveShipping>'

  6) Spree::Calculator::Shipping service_name should return description when not defined
     Failure/Error: order = FactoryGirl.create(:order_with_line_items, :ship_address => address, :line_items_count => 2)
     ArgumentError:
       wrong number of arguments (1 for 0)
     # ./spec/models/active_shipping_calculator_spec.rb:11:in `block (2 levels) in <module:ActiveShipping>'
```

Am I doing something wrong here?

**Comment by jspizziri** [comment]
Is your PR in reference to this issue? or are they unrelated.

**Comment by brchristian** [comment]
@jspizziri PR is related in that `bundle install / be rake test_app` were failing in the `2-3-stable` branch until I updated the dependencies to point to Rails 4.1 (which Spree 2.3 uses) rather than 4.0.

That got me as far as a successful Bundle installation and a successful `be rake test_app`, at which point my specs ran but failed with the output above.

**Comment by jspizziri** [comment]
https://github.com/spree-contrib/spree_active_shipping/pull/214

**Comment by jspizziri** [comment]
Fixed by #214
